introduction-to-datasets.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
```

Comment:
_Like_ can be interpreted to mean _identical_. _Similar to_, while redundant, is more clear.

_But_ the conjunction is not the same as _not only...but also_, a correlative conjunction; this is fine grammatically and conveys the proper meaning.